.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
+
+make serve-local &
+
+curl --retry-connrefused \
+    --retry 5 \
+    --retry-delay 20 \
+    http://127.0.0.1:5000 --fail
+
+var=$?
```

Comment:
This varname is not explicit.
```suggestion
result=$?
```